gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -1,24 +1,30 @@
 ---
 --
+-- Global temporary replacements for targetid.lua
+ParT = LANG.GetParamTranslation
+TryT = LANG.TryTranslation
 
+key_params = {
+    usekey = Key("+use", "USE"),
+    walkkey = Key("+walk", "WALK")
+}
+
```

Comment:
These will be the placeholders until I found a better solution.
Otherwise this refactoring works now and is "finished".